src/rpc/rawtransaction.cpp

```diff
@@ -899,7 +899,7 @@ UniValue sendrawtransaction(const JSONRPCRequest& request)
         // push to local node and sync with wallets
         CValidationState state;
         bool fMissingInputs;
-        if (!AcceptToMemoryPool(mempool, state, tx, fLimitFree, &fMissingInputs, false, nMaxRawTxFee)) {
+        if (!AcceptToMemoryPool(mempool, state, std::move(tx), fLimitFree, &fMissingInputs, false, nMaxRawTxFee)) {
```

Comment:
I believe this `std::move` is unnecessary. ATMP takes a const reference to tx, so casting tx to an rvalue here doesn't do anything (ATMP must copy from tx if it wants to retain shared ownership).